docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -140,6 +140,14 @@ \section{Failure Mode and Effect Analysis}
       failure & detailed logs of all views and edits to documents
       from within the system & a. Allow administrator to adjust
       permissions & SR-AR2 \newline SR-AR3 \newline SR-IR1 & H2-2 \\
+      \cline{2-8}
+      & User unable to complete Two-Factor Authentication & User
+      unable to access application & a. User name and email not in
```

Comment:
this doesn't sound like a failure mode. If the user doesn't exist, then I wouldn't want them to have access to our system.